### PR TITLE
Harmonization of road mark colors with OpenDRIVE

### PR DESCRIPTION
**Pull request by MarcelOliver**
As suggested in #504:
**1. Color orange is added to road mark colors**
**2. Color violet is marked as deprecated**

**Some questions to ask**:
What is this change? --> only extension
What does it fix? --> harmonisation with OpenDrive
Is this a bug fix or a feature? Does it break any existing functionality or force me to update to a new version?
How has it been tested? --> only extension

#### Take this checklist as orientation for yourself, if this PR is ready for the Change Control Board:
- [x] My suggestion follows the [style and contributors guidelines](https://opensimulationinterface.github.io/osi-documentation/open-simulation-interface/doc/howtocontribute.html).
- [x] I have taken care about the [documentation](https://opensimulationinterface.github.io/osi-documentation/open-simulation-interface/doc/commenting.html).
- [ ] I have done the [DCO signoff](https://opensimulationinterface.github.io/osi-documentation/open-simulation-interface/doc/howtocontribute.html#developer-certification-of-origin-dco).
- [ ] My changes generate no errors when passing CI tests. 
- [ ] I have successfully implemented and tested my fix/feature locally.
- [ ] Appropriate reviewer(s) are assigned.

If you can’t check all of them, please explain why.
If all boxes are checked or commented and you have achieved at least one positive review, you can assign the label ReadyForCCBReview!

### CODE REVIEW

**Review by HendrikAmelunxen — APPROVED** [review]
Looks good to me

**Comment by HendrikAmelunxen** [comment]
Harmonisation 22.04.:
Change comment regarding to Architecture group: 

\attention Deprecated: info 

"info" should contain the following: Provide information about the deprecated message.What needs to be changed/code adaption etc..e.g. Use assigned_lane_id in MovingObjectClassification instead.Provide information on when it will be deprecated.Specify the release version (usually, a major release )

**Comment by HendrikAmelunxen** [comment]
Harmonisation 22.04.:
Extend Roadmarkings and Laneboudnaries by "orange"; "violet" stays; no deprecated comment
=> For OSI 4.0: centralize color in osi_common

**Comment by MarcelOliver** [comment]
At the moment, color is defined in four different files (**osi_lane**, **osi_object**, **osi_featuredata** and **osi_roadmarking**) as shown here:

**osi_lane.proto:**
- COLOR_UNKNOWN = 0;
- COLOR_OTHER = 1;
- COLOR_NONE = 2;
- COLOR_WHITE = 3;
- COLOR_YELLOW = 4;
- COLOR_RED = 5;
- COLOR_BLUE = 6;
- COLOR_GREEN = 7;
- COLOR_VIOLET = 8;
- COLOR_ORANGE = 9;

		
**osi_object.proto:**
- COLOR_UNKNOWN = 0;
- COLOR_OTHER = 1;
- COLOR_YELLOW = 2;
- COLOR_GREEN = 3;
- COLOR_BLUE = 4;
- COLOR_VIOLET = 5;
- COLOR_RED = 6;
- COLOR_ORANGE = 7;
- COLOR_BLACK = 8;
- COLOR_GREY = 9;
- COLOR_WHITE = 10;

**osi_featuredata.proto:**
- COLOR_UNKNOWN = 0;
- COLOR_OTHER = 1;
- COLOR_BLACK = 2;
- COLOR_GREY = 3;
- COLOR_WHITE = 4;
- COLOR_YELLOW = 5;
- COLOR_ORANGE = 6;
- COLOR_RED = 7;
- COLOR_VIOLET = 8;
- COLOR_BLUE = 9;
- COLOR_GREEN = 10;
- COLOR_REFLECTIVE = 11;	

	
**osi_roadmarking.proto:**
- COLOR_UNKNOWN = 0;
- COLOR_OTHER = 1;
- COLOR_WHITE = 2;
- COLOR_YELLOW = 3;
- COLOR_BLUE = 5;
- COLOR_RED = 6;
- COLOR_GREEN = 7;
- COLOR_VIOLET = 8;	
- COLOR_ORANGE = 9;

**--> For OSI 4.0: centralize color in osi_common**

**Comment by HendrikAmelunxen** [comment]
This topic is addressed in issue 512 [https://github.com/OpenSimulationInterface/open-simulation-interface/issues/512](url)

File: osi_lane.proto [hunk 1]
```diff
@@ -993,6 +993,10 @@ message LaneBoundary
             // Marking with violet color.
             //
             COLOR_VIOLET = 8;
+			
+			// Marking with orange color.
```
Comment (kmeids) [inline osi_lane.proto:997]: Probably White space issue.

**Comment by kmeids** [comment]
**Output CCB 12.05.2021:**
1. @pmai will fix the DCO, white space issues and merge.

**Pull request merged**